_test/test_dnd_class/test_dnd_cut.m

```diff
@@ -3,52 +3,66 @@
     % Check various dnd cuts comparing them with the reference cuts stored
     % earlier
     %
-    
-    
+
     properties
         dnd_file_2d_name = 'dnd_2d.sqw';
         d2d_obj;
     end
-    
+
     methods
-        
+
         %The above can now be read into the test routine directly.
         function obj=test_dnd_cut(varargin)
             test_ref_data = fullfile(fileparts(mfilename('fullpath')),'test_dnd_cut.mat');
 
             argi = [varargin,test_ref_data];
 
-            obj = obj@TestCaseWithSave(argi{:});            
+            obj = obj@TestCaseWithSave(argi{:});
             hp = horace_paths();
             dnd_2d_fullpath = fullfile(hp.test_common,obj.dnd_file_2d_name);
-            %hor_root = horace_root();            %            
+            %hor_root = horace_root();            %
             %dnd_2d_fullpath = fullfile(hor_root,'_test/common_data',...
-            %    obj.dnd_file_2d_name);            
+            %    obj.dnd_file_2d_name);
 
             obj.d2d_obj = read_dnd(dnd_2d_fullpath);
-            obj.save();            
+            obj.save();
+        end        
+        % tests
+        function test_2D_to2D_cut_with_spher_proj(obj)
+            ws = warning('off','HORACE:runtime_error');
+            clOb = onCleanup(@()warning(ws));
+            
+            proj = spher_proj();
+            cut_range = obj.d2d_obj.targ_range(proj);
+            w2 = cut(obj.d2d_obj,proj,[1,0.1,2], ...
+                [113,114],[cut_range(1,3),0.1,cut_range(2,3)],[-0.25,0.25]);
+            skipTest('#983 does this spherical projection work correctly?')
```

Comment:
if this works formally we should not be skipping this test, we should be putting in a ticket for the scientific validation

Reply:
this what it is done. I wrote the ticket, all working parts of the test are running but the result is questionable.